node/server.go

```diff
@@ -338,6 +339,13 @@ func Run(ctx context.Context, opts Options) error {
 		fmt.Printf("==> Connected to Filecoin RPC at %s\n", opts.FilEndpoint)
 	}
 
+	if reflect.TypeOf(nd.metrics) != nil {
```

Comment:
```suggestion
	if nd.metrics != nil {
```

Reply:
What you could do is create an empty Metrics interface so you don't need to check every time if nd.metrics is nil

Reply:
So you'd do:
```go
type NullMetrics struct {}

func (*NullMetrics) Record(...) {}

etc
```
Then when you initialize it you could have a contructor `NewMetrics` that either returns an InfluxDB if the params are there of a NullMetrics